packages/api/test/codegen/languages/typescript/index.test.ts

```diff
@@ -97,22 +97,16 @@ function assertSDKFixture(file: string, fixture: string) {
 
 describe('typescript', () => {
   beforeEach(() => {
-    vi.setSystemTime(new Date('2023'));
```

Comment:
Had to change this to a fuller date because `new Date('2023').getFullYear()` returns 2022.